HW2/src/test/java/com/example/hw2/controller/AuthorControllerTest.java

```diff
@@ -0,0 +1,68 @@
+package com.example.hw2.controller;
+
+import com.example.hw2.servise.AuthorService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(AuthorController.class)
+@AutoConfigureMockMvc
+class AuthorControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private AuthorService service;
+
+    @Test
+    public void getAll() throws Exception {
```

Comment:
Prefer to use more verbose and descriptive name:

`<which-method/api>_should<expected-behaviour>_<use-case/condition>`

+ use `@DisplayName("description here")` to describe test case.